File: frontend/src/app/dto/reponses/transaction-recente.dto.ts
export interface TransactionRecente {
    codeTransaction: string;
    nomAbonne: string;
    prenomAbonne: string;
    numeroCompteur: string;
    montantVerse: number;
    quantiteKwh: number;
    dateTransaction: Date;
    statut: string;
    caissiere: string;
}

File: backend/src/main/java/com/sonabel/venkcash/dto/TransactionRecenteDTO.java
package com.sonabel.venkcash.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
@AllArgsConstructor
public class TransactionRecenteDTO {
    private String codeTransaction;
    private String nomAbonne;
    private String prenomAbonne;
    private String numeroCompteur;
    private BigDecimal montantVerse;
    private BigDecimal quantiteKwh;
    private LocalDateTime dateTransaction;
    private String statut;
    private String caissiere;
}
